src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -56,7 +56,7 @@
 public class Dataverse extends DvObjectContainer {
 
     public enum DataverseType {
-        RESEARCHERS, RESEARCH_PROJECTS, JOURNALS, ORGANIZATIONS_INSTITUTIONS, TEACHING_COURSES, UNCATEGORIZED
+        RESEARCHERS, RESEARCH_PROJECTS, JOURNALS, ORGANIZATIONS_INSTITUTIONS, TEACHING_COURSES, UNCATEGORIZED, LABORATORY, RESEARCH_GROUP
```

Comment:
Sounds good to me!! :+1

Eleni Castro
Research Coordinator, IQSS
Harvard University
On Apr 12, 2016 3:32 PM, "Philip Durbin" <notifications@github.com> wrote:

> In src/main/java/edu/harvard/iq/dataverse/Dataverse.java
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_IQSS_dataverse_pull_3070-23discussion-5Fr59438771&d=CwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=OiTcIL_jSSCUW82Gf-OSkgnZLbG2Yt87eT87BdLiP54&m=Ex7UsjgHI_AvPsnkhYWuwZIRRhxMegtBLolLmXUtIaY&s=eNz3mzzCL2E1i2derzbobHpktZ_cIF6vwf1XJcvYFLQ&e=>
> :
>
> > @@ -56,7 +56,7 @@
> >  public class Dataverse extends DvObjectContainer {
> >
> >      public enum DataverseType {
> > -        RESEARCHERS, RESEARCH_PROJECTS, JOURNALS, ORGANIZATIONS_INSTITUTIONS, TEACHING_COURSES, UNCATEGORIZED
> > +        RESEARCHERS, RESEARCH_PROJECTS, JOURNALS, ORGANIZATIONS_INSTITUTIONS, TEACHING_COURSES, UNCATEGORIZED, LABORATORY, RESEARCH_GROUP
>
> While we're at it should we add Department? See Dataverse Category: Add
> Department #2829
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_IQSS_dataverse_issues_2829&d=CwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=OiTcIL_jSSCUW82Gf-OSkgnZLbG2Yt87eT87BdLiP54&m=Ex7UsjgHI_AvPsnkhYWuwZIRRhxMegtBLolLmXUtIaY&s=2AnjgM9BmVR0EyFoqXUZd-NpqY-mPR3RbvlZtHj-KqE&e=>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_IQSS_dataverse_pull_3070_files_2b7232d72e5221514c5e4455922d81dd4becf6ca-23r59438771&d=CwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=OiTcIL_jSSCUW82Gf-OSkgnZLbG2Yt87eT87BdLiP54&m=Ex7UsjgHI_AvPsnkhYWuwZIRRhxMegtBLolLmXUtIaY&s=8gAJ-zGMGNsuTvrNZZ0NuRDJdsOEmPB0kjuqsRaCruY&e=>
>